etc/kubernetes/helm/enterprise-gateway/templates/daemonset.yaml

```diff
@@ -38,14 +38,26 @@ spec:
           value: {{ .Values.kip.defaultContainerRegistry }}
         # Optional authorization token passed in all requests (should match EG_AUTH_TOKEN)
         {{- if .Values.authToken }}
-        - name: KIP_AUTH_TOKEN   
+        - name: KIP_AUTH_TOKEN
           value: {{ .Values.authToken }}
         {{- end }}
         volumeMounts:
         - name: cri-socket
           mountPath: !!str {{ .Values.kip.criSocket }}  # see env KIP_CRI_ENDPOINT
+        {{- if .Values.kip.imagePullSecret }}
+        - name: dockerconfig
+          mountPath: "/root/.docker"
```

Comment:
I see that we're running this image as `root` (hadn't really noticed that before).  Does the mount have to be in the container user's HOME dir? (sorry, it seems the docs imply this, but I don't know if that's just an example). 
I wonder if this particular value should be templated as well via something like `.Values.kip.pullSecretMountPath` with this as the default value?

Reply:
this might help : https://www.devopsschool.com/blog/how-to-configure-docker-client-with-a-configuration-file/

Reply:
Thanks @rahul26goyal.  It looks like docker client supports a `DOCKER_CONFIG` env and I wonder if we should let that value dictate the `mountPath`?